vscode/src/status.ts

```diff
@@ -109,6 +109,12 @@ export class ServerStatus extends StatusItem {
         break;
       }
     }
+
+    if (workspace.lspClient.degraded) {
```

Comment:
Do you have an expectation for how this should behave in conjunction with the Start/Stop/Restart commands?

Reply:
Should this be put under the `State.Running` case instead?

Reply:
Yeah, I think that makes sense. Updated the code to only append `(degraded)` and change the status.